packages/xo-web/src/xo-app/home/index.js

```diff
@@ -1106,6 +1142,49 @@ export default class Home extends Component {
               </div>
             ) : (
               <div>
+                {(type === 'VM' || type === 'host') && (
+                  <Tooltip
+                    content={
+                      selectedPowerState == null
+                        ? undefined
+                        : _(`${selectedPowerState}Item`, { type: `${type}s` })
```

Comment:
Do we need to show the object type here? Can't we just say "Running", "Halted", ...?